console.rst

```diff
@@ -146,6 +146,8 @@ the console::
         // outputs a message without adding a "\n" at the end of the line
         $output->write('You are about to ');
         $output->write('create a user.');
+        
+        return 0;
```

Comment:
How does your `execute()` method looks like?

a) `protected function execute(InputInterface $input, OutputInterface $output)`
b) `protected function execute(InputInterface $input, OutputInterface $output): int`

I guess `b)`, but then its your return type and not the Symfony one.
Symfony 5.0 does not have a return type:
https://github.com/symfony/symfony/blob/5.0/src/Symfony/Component/Console/Command/Command.php#L159

Reply:
I would say `a)` because Symfony throw it self the error : https://github.com/symfony/symfony/blob/5.0/src/Symfony/Component/Console/Command/Command.php#L258

Reply:
Oh interesting... You are right, didn't check it this way 🤔

Reply:
yep, @maxhelias is right, it's `a)`, just copy past current docs on a fresh 5.0 to reproduce